hummingbot/connector/exchange/ndax/ndax_websocket_adaptor.py

```diff
@@ -81,17 +81,20 @@ async def recv(self):
     async def iter_messages(self) -> AsyncIterable[str]:
         try:
             while True:
-                msg: str = await asyncio.wait_for(self.recv(), timeout=self.MESSAGE_TIMEOUT)
-                yield msg
-        except asyncio.TimeoutError:
-            await asyncio.wait_for(
-                self.send_request(CONSTANTS.WS_PING_REQUEST, payload={}),
-                timeout=self.PING_TIMEOUT
-            )
+                try:
+                    msg: str = await asyncio.wait_for(self.recv(), timeout=self.MESSAGE_TIMEOUT)
+                    yield msg
+                except asyncio.TimeoutError:
+                    await asyncio.wait_for(
+                        self.send_request(CONSTANTS.WS_PING_REQUEST, payload={}),
+                        timeout=self.PING_TIMEOUT
+                    )
+                except Exception as e:
```

Comment:
No need to capture the exception if we will just pass it (or raise it again)

Reply:
Yea I overlooked this. Yea my intention was to capture this error from in the `NdaxAPIOrderBookDataSource` class, where `iter_messages()` is called.